docs/xml/elements/hours_open.rst

```diff
@@ -1,57 +1,103 @@
 HoursOpen
 =========
 
-.. todo::
+A structured way of describing the days and hours that a place such as a
+:doc:`Office <office>` or :doc:`PollingLocation <polling_location>` is open, or
+that an event such as an :doc:`Election <election>` is happening.
 
-   Document HoursOpen & Schedule
-
-+----------+--------------------+------------+----------+----------------------+----------------------------+
-| Tag      | Data Type          | Required?  | Repeats? |Description           |Error Handling              |
-|          |                    |            |          |                      |                            |
-+==========+====================+============+==========+======================+============================+
-| Schedule | Schedule           | Optional   | Repeats  |                      |                            |
-+----------+--------------------+------------+----------+----------------------+----------------------------+
++----------+--------------------+------------+----------+-----------------------+-----------------------------------+
+| Tag      | Data Type          | Required?  | Repeats? |Description            |Error Handling                     |
+|          |                    |            |          |                       |                                   |
++==========+====================+============+==========+=======================+===================================+
+| Schedule | Schedule           |**Required**| Repeats  |Defines a block of days| At least one valid `Schedule`_    |
```

Comment:
Does the \`Schedule\`_ link work? Shouldn't it be \`HoursOpen.Schedule\`_? The heading could change to just \`Schedule\`_, which will fit better in the sidebar navigation.